src/OpenLoco/src/Windows/ObjectSelectionWindow.cpp

```diff
@@ -485,24 +482,15 @@ namespace OpenLoco::Ui::Windows::ObjectSelectionWindow
         return window;
     }
 
-    static void switchTab(Window& self, ObjectType newTab);
+    static void switchPrimaryTab(Window& self, uint8_t tabIndex);
+    static void switchTabByObjectType(Window& self, ObjectType objectType);
 
     Window& openInTab(ObjectType objectType)
     {
         auto& window = *open();
-        auto& info = _tabDisplayInfo[enumValue(objectType)];
-
-        if ((info.flags & ObjectTabFlags::alwaysHidden) != ObjectTabFlags::none)
-        {
-            window.var_856 = enumValue(FilterLevel::expert);
-        }
-        else
-        {
-            window.var_856 = enumValue(FilterLevel::advanced);
-        }
-
+        window.var_856 = enumValue(FilterLevel::advanced);
```

Comment:
`var_856` - sounds like we know what it does and we can give it a name?

Reply:
Unfortunately it is used by different windows for different purposes... Though we could perhaps make a union for it?